pkg/maps/lxcmap/lxcmap.go

```diff
@@ -83,13 +85,58 @@ const (
 )
 
 // EndpointFrontend is the interface to implement for an object to synchronize
-// with the endpoint BPF map
+// with the endpoint BPF map.
 type EndpointFrontend interface {
-	// GetBPFKeys must return a slice of EndpointKey which all represent the endpoint
-	GetBPFKeys() []*EndpointKey
+	LXCMac() mac.MAC
+	GetNodeMAC() mac.MAC
+	GetIfIndex() int
+	GetID() uint64
+	IPv4Address() addressing.CiliumIPv4
+	IPv6Address() addressing.CiliumIPv6
+}
+
+// GetBPFKeys returns all keys which should represent this endpoint in the BPF
+// endpoints map
+func GetBPFKeys(e EndpointFrontend) []*EndpointKey {
```

Comment:
small nit: the older functions that take EndpointFrontend seemed to name the parameter `f`